rotkehlchen/db/upgrades/v46_v47.py

```diff
@@ -206,4 +206,24 @@ def _reset_gemini_events(write_cursor: 'DBCursor') -> None:
             ),
         )
 
+    @progress_step(description='Reset bybit events and cache')
+    def _reset_bybit_events(write_cursor: 'DBCursor') -> None:
+        write_cursor.execute(
```

Comment:
can't we just combine it with the previous step? You are repeting code here that could be a for loop unless I'm missing something